pkg/hive/mock/mock.go

```diff
@@ -0,0 +1,35 @@
+package mock
```

Comment:
Package mock should provide mock implementation of the Hive protocol. Here are only test helpers for hive tests, which can go there. If they should be here, then suffix Mock can be dropped from all identifiers.

Reply:
I will move them to `hive_tests.go` then.